### PR TITLE
Add edit predicate button and initial browser test

### PR DESCRIPTION
**Pull request by carolinedanzi**
### Description
1. Adds a button on the block edit view to go to the Edit Visibility Condition page
2. Sets up a basic browser test that adds a predicate and checks that it was added

### Checklist
- [x] Created tests which fail without the change (if possible)
- [ ] Extended the README / documentation, if necessary

### Issue(s)
#322

### CODE REVIEW

File: browser-test/src/admin_predicates.test.ts [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import { AdminPredicates, AdminPrograms, AdminQuestions, endSession, loginAsAdmin, logout, startSession } from './support'
+
+describe('create and edit predicates', () => {
```
Comment (natsid) [inline browser-test/src/admin_predicates.test.ts:3]: praise: Thank you so much for adding this!

File: universal-application-tool-0.0.1/app/views/admin/programs/ProgramBlockEditView.java [hunk 2]
```diff
@@ -281,7 +281,6 @@ private ContainerTag blockMoveButtons(
   private ContainerTag blockEditPanel(
       ProgramDefinition program,
       BlockDefinition blockDefinition,
-      long blockId,
```
Comment (natsid) [inline universal-application-tool-0.0.1/app/views/admin/programs/ProgramBlockEditView.java:284]: praise: Thanks for this cleanup.

File: universal-application-tool-0.0.1/app/views/admin/programs/ProgramBlockEditView.java [hunk 3]
```diff
@@ -355,16 +357,25 @@ private ContainerTag blockEditPanel(
...
         .withClasses(Styles.M_4)
         .with(
             div("Visibility Condition").withClasses(Styles.TEXT_LG, Styles.FONT_BOLD, Styles.PY_2))
+        .with(div(currentBlockStatus).withClasses(Styles.TEXT_LG, Styles.MAX_W_PROSE))
         .with(
-            div(predicate.toDisplayString(blockName, questions))
-                .withClasses(Styles.TEXT_LG, Styles.MAX_W_PROSE));
+            redirectButton(
+                ReferenceClasses.EDIT_PREDICATE_BUTTON,
+                "Edit Visibility Condition",
```
Comment (natsid) [inline universal-application-tool-0.0.1/app/views/admin/programs/ProgramBlockEditView.java:377]: nit: Update this to use Sentence case, like "Edit visibility condition" - same with "Visibility condition" above. It's a small thing, but good for keeping our casing consistent across the site. :)

File: browser-test/src/support/admin_predicates.ts [hunk 4]
```diff
@@ -0,0 +1,22 @@
+import { Page } from 'playwright'
+
+export class AdminPredicates {
```
Comment (natsid) [inline browser-test/src/support/admin_predicates.ts:3]: praise: And the helpers!! 🎉

**Review by natsid — APPROVED** [review]
Exciting! Nice work.

File: universal-application-tool-0.0.1/app/views/admin/programs/ProgramBlockEditView.java (hunk 3 above)
Comment (carolinedanzi) [reply universal-application-tool-0.0.1/app/views/admin/programs/ProgramBlockEditView.java:377]: Thanks so much for catching that!

**Pull request merged**